### ISSUE TITLE
Issue with removing all versions of a second language

### ISSUE THREAD

**Issue by MarcinOkon** [sergeyshushlyapin/Sitecore.FakeDb#121]
Hi
I have found an issue when trying to use
```c#
item.Versions.RemoveAll(true)
``` 
on an item with multiple languages and versions

It seems fine when the english version has 2 versions
```c#
public void RemoveVersionsTestWith2EnVersions()
{
	////Arrange
	using (var db = new Db
	{
		new DbItem("Item")
		{
			new DbField("Title")
			{
				{ "en", "Hello!" },
				{ "de", "Servus!" }
			}
		}
	})
	{
		Item itemEn = db.GetItem("/sitecore/content/Item", "en");
		Item itemDe = db.GetItem("/sitecore/content/Item", "de");
		itemEn.Versions.AddVersion();

		Assert.AreEqual(2, itemEn.Versions.Count);
		Assert.AreEqual(1, itemDe.Versions.Count);

		//Act
		itemEn.Versions.RemoveAll(true);

		//Assert
		Assert.AreEqual(0, itemEn.Versions.Count);
		Assert.AreEqual(0, itemDe.Versions.Count);
	}
}
```

but fails when a different language has 2 versions
```c#
[Test]
public void RemoveVersionsTestWith2DeVersions()
{
	////Arrange
	using (var db = new Db
	{
		new DbItem("Item")
		{
			new DbField("Title")
			{
				{ "en", "Hello!" },
				{ "de", "Servus!" }
			}
		}
	})
	{
		Item itemEn = db.GetItem("/sitecore/content/Item", "en");
		Item itemDe = db.GetItem("/sitecore/content/Item", "de");
		itemDe.Versions.AddVersion();

		Assert.AreEqual(1, itemEn.Versions.Count);
		Assert.AreEqual(2, itemDe.Versions.Count);

		//Act
		itemEn.Versions.RemoveAll(true); //throws a 'System.InvalidOperationException : Sequence contains no elements'

		//Assert
		Assert.AreEqual(0, itemEn.Versions.Count);
		Assert.AreEqual(0, itemDe.Versions.Count);
	}
}
```

**Comment by sergeyshushlyapin** [comment]
I'll take a look, thanks for reporting, @MarcinOkon.

**Comment by sergeyshushlyapin** [comment]
Fixed in [v0.37.0](https://www.nuget.org/packages/Sitecore.FakeDb/).

**Comment by MarcinOkon** [comment]
thanks for the fast fix
